oauth2client/contrib/locked_file.py

```diff
@@ -347,10 +189,15 @@ def __init__(self, filename, mode, fallback_mode, use_native_locking=True):
         """
         opener = None
         if not opener and use_native_locking:
-            if _Win32Opener:
+            try:
+                from oauth2client.contrib._win32_opener import _Win32Opener
```

Comment:
Not sure what you mean.

On Wed, Apr 6, 2016 at 2:30 PM Danny Hermes <notifications@github.com>
wrote:

> In oauth2client/contrib/locked_file.py
> <https://github.com/google/oauth2client/pull/478#discussion_r58785429>:
>
> > @@ -347,10 +189,15 @@ def __init__(self, filename, mode, fallback_mode, use_native_locking=True):
> >          """
> >          opener = None
> >          if not opener and use_native_locking:
> > -            if _Win32Opener:
> > +            try:
> > +                from oauth2client.contrib._win32_opener import _Win32Opener
>
> Ahhh that is a problem. Thoughts on just defining mix-ins in the platform
> specific modules?
>
> —
> You are receiving this because you were mentioned.
>
> Reply to this email directly or view it on GitHub
> <https://github.com/google/oauth2client/pull/478/files/72cd28b5322528c950aa54b97e688330f2a12867#r58785429>
>